boards/arm/nrf9160dk_nrf9160/nrf9160dk_nrf9160_partition_conf.dts

```diff
@@ -39,20 +39,22 @@
 
 /* Default SRAM planning when building for nRF9160 with
  * ARM TrustZone-M support
- * - Lowest 64 kB SRAM allocated to Secure image (sram0_s).
- * - 64 kB SRAM reserved for and used by the BSD socket
- *   library (sram0_bsd).
+ * - Lowest 88 kB SRAM allocated to Secure image (sram0_s).
+ * - 40 kB SRAM reserved for and used by the modem library
+ *   (sram0_modem). This memory is Non-Secure.
  * - Upper 128 kB allocated to Non-Secure image (sram0_ns).
+ *   When build with TF-M, both sram0_modem and sram0_ns
+ *   are allocated to the Non-Secure image.
  */
 
 &sram0_s {
-	reg = <0x20000000 DT_SIZE_K(64)>;
+	reg = <0x20000000 DT_SIZE_K(88)>;
 };
 
-&sram0_bsd {
-	reg = <0x20010000 DT_SIZE_K(64)>;
+&sram0_modem {
+	reg = <0x20010000 DT_SIZE_K(40)>;
 };
 
 &sram0_ns {
-	reg = <0x20020000 DT_SIZE_K(128)>;
+	reg = <0x20016000 DT_SIZE_K(168)>;
```

Comment:
This should stay as it was.

Reply:
I think this is not supported, currently in upstream TF-M @anangl

Reply:
Ah, right, I missed that "detail". Then I guess this line should be updated accordingly: https://github.com/zephyrproject-rtos/zephyr/blob/31d54df3ea7d503619c5e8f4c556711984955b0b/boards/arm/nrf9160dk_nrf9160/nrf9160dk_nrf9160_common.dts#L221

Reply:
Edit: what i mean is that TF-M partition layout is not aware (At the moment) of the modem shared memory area, so I was thinking that by default Zephyr samples could make use of all existing non-secure SRAM.

But it should be fine to change this back to what it was.

Reply:
done, @anangl